case-summarization/guardails/guardrails.yaml

```diff
@@ -0,0 +1,71 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: AWS Bedrock Guardrail Template Stack
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+            default: 'Amazon Bedrock Guardrail parameters'
+        Parameters:
+          - BlockedInputMessage
+          - BlockedOutputMessage
+      - Label:
+            default: 'Technical parameters'
+        Parameters:
+          - CFDataName
+          - ResourcePrefix
+
+Parameters:
+  CFDataName:
+    Type: String
+    Description: The name of what this cf is doing.
+    Default: bedrock-guardrails
+  ResourcePrefix:
+    Type: String
+    Description: This prefix will be placed in front of all resources created. Note you may wish to add a dash at the end to make more readable (e.g. 'prefix-'). This parameter CANNOT BE UPDATED. Delete and re-create stack if needed an update.
+    Default: "CID-DC-"
+  BlockedInputMessage:
+    Type: String
+    Description: Message to return when the Amazon Bedrock Guardrail blocks a prompt.
+    MaxLength: 500
+    Default: 'Your input was blocked by the content policy.'
+  BlockedOutputMessage:
+    Type: String
+    Description: Message to return when the Amazon Bedrock Guardrail blocks a model response.
+    MaxLength: 500
+    Default: 'The model response was blocked by the content policy.'
+
+Resources:
+  BedrockGuardrail:
```

Comment:
Perhaps consider adding sensitive content filters as a starter too.

Reply:
If sensitive data are in data they will be in dashboard. So they may appear in summarizartion field as well. 

very good point that we could do in v2 is to make safer version of cases filtering out all sensitive info so that it could be shared wider with more confidence. 

this is a good idea but let's keep it for v2

cc @schniber